### PR TITLE
Cover art in Now Playing Bar is now Responsive

### PR DESCRIPTION
**Pull request by elpideus**
## ✨ Lucid Theme Enhancement ✨

**Tell us about your changes!** 📝

- **TL;DR:** (Cover art in Now Playing Bar is now Responsive)
- **Deep Dive:** (Improved responsiveness of the Cover Art in the Now Playing Bar so if the Bar is resized to lower or higher values the Cover Art decreases and increases its changes automatically)

**What kind of change did you make?** 🚀

- [x] 🐛 **Bug fix:** Crushed a bug! This change fixes an existing issue without impacting other functionalities.
- [ ] ✨ **New feature:** Shiny and new! This adds a cool new functionality to the theme.
- [ ] 💥 **Breaking change:** Watch out! This fix or feature might cause some existing functionalities to behave differently.
- [ ] 📚 **Documentation update:** Making the docs even better! This improves the documentation for easier understanding.
- [ ] 🌈 **Other:** (Describe your awesome change here!)

**Why did you make this change?** 🤔

- **The Problem:** (Lowering the bar height to lower values than 60 resulted in the Cover Art looking weird)
- **The Solution:** (Added dynamic size to the Cover Art)

**Did you test it out?** 🧪

- **Testing Steps:** (Tested different values for the Now Playing Bar and it works well if the values are not lower than 23)

**Show it off! (Optional)** 

image

image

**Final Check!** ✅

- [x] 👌 **Code Style:** My code follows the project's style guidelines.
- [ ] 📝 **Documentation:** I've updated the documentation to reflect my changes (if needed).
- [ ] 🧪 **Tests Passing:** I've added tests for my changes (where applicable) and made sure all tests pass successfully.

**Anything else?** 💬

I will probably update the theme in the future to support even lower values for the bar while mantaining the design language of the theme.

### CODE REVIEW

**Comment by sanoojes** [comment]
image

Please update the SCSS as requested and compile it into user.css by running npm run compile-sass. Please also check the package.json file.
